### DISCUSSION TITLE
Unable to download model

### DISCUSSION THREAD

**Discussion by prurigro** [developersharif/bgremover-app#9] [General]
**Describe the bug**

When starting the app or selecting the download model option, a popup appears with the following error:

```
 An error occurred while downloading the model file: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:997)>
```

**To Reproduce**

1. Download the 2.0 release for Linux
2. Start the app

**Expected behavior**

The app starts and the model downloads correctly, or directs the user to where the model is and has them manually download it.

**Screenshots**

![error](https://user-images.githubusercontent.com/1149238/228031982-7ee3eb2c-17a7-4294-8cc2-5a44c0f3c68d.png)

**Desktop (please complete the following information):**
 - OS: Linux
 - Version: [V2.0](https://github.com/developersharif/bgremover-app/releases/tag/v2.0.0)

**Comment by developersharif** [comment]
Update your SSL certificates:
You can try updating your SSL certificates to ensure that they are up-to-date and valid. This can typically be done by running the following command in your terminal:

```bash
sudo update-ca-certificates --fresh
```
Set the SSL_CERT_DIR environment variable:
You can try setting the SSL_CERT_DIR environment variable to the path where your SSL certificates are located. This can typically be done by running the following command in your terminal:
```bash
export SSL_CERT_DIR=/etc/ssl/certs
```

**Comment by developersharif** [comment]
> **Describe the bug**
> 
> When starting the app or selecting the download model option, a popup appears with the following error:
> 
> ```
>  An error occurred while downloading the model file: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:997)>
> ```
> 
> **To Reproduce**
> 
> 1. Download the 2.0 release for Linux
> 2. Start the app
> 
> **Expected behavior**
> 
> The app starts and the model downloads correctly, or directs the user to where the model is and has them manually download it.
> 
> **Screenshots**
> 
> ![error](https://user-images.githubusercontent.com/1149238/228031982-7ee3eb2c-17a7-4294-8cc2-5a44c0f3c68d.png)
> 
> **Desktop (please complete the following information):**
> 
> * OS: Linux
> * Version: [V2.0](https://github.com/developersharif/bgremover-app/releases/tag/v2.0.0)

Or run the script  https://raw.githubusercontent.com/developersharif/bgremover-app/main/linux-model-download.sh

**Comment by prurigro** [comment]
Hey, I'm not sure why the app can't verify the ssl certificate, but I'm able to download it directly, thanks!
